### ISSUE TITLE
User Cannot able to invoke preferences tab items "general/updates/feedback" using keyboard.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#389]
### Steps to Reproduce

1. Launch Xamarin Workbooks application. This will invokes "Create a new Workbook" window.
2. Navigate to Tools -> Preferences and invoke it. This will opens "General tab" of Preferences
3. Try to navigate to "general/updates/feedback" from one to another tabs using keyboard and invoke them.

### Expected Behavior
Should be able to invoke them using keyboard.

PositionInSet and Size of set property should be defined for tab items and it should also be read by screen-reader
### Actual Behavior
User Cannot be able to invoke preferences tabs "general/updates/feedback" using keyboard.
User cannot set focus on "updates/feedback" tabs and also cannot able to navigate to other tabs(updates/feedback) from general tab.

Note:
PositionInSet and Size of set property  for tab items is not being announced by Screenreader

Narrator Behavior:
Please refer the below screenshot.
![screenshot 189](https://user-images.githubusercontent.com/22956130/40766336-c473611e-64cc-11e8-94f1-acd254b765bf.png)

NVD Behavior:
Please refer the below screenshot.
![screenshot 190](https://user-images.githubusercontent.com/22956130/40766341-cae233b8-64cc-11e8-90f0-87e894a92a7d.png)

Inspect Behavior:
Please refer the below screenshot.
![screenshot 191](https://user-images.githubusercontent.com/22956130/40766348-d081fa4c-64cc-11e8-82d0-5debc5f2987e.png)

### User Impact
Users need to be able to navigate an application and interact with the UI elements by using a keyboard alone. Users who are dependent on keyboard navigation include those with visual impairments who rely on screen readers and those with dexterity or mobility impairments who cannot use a mouse. 
### Screenshots:
![issue 5](https://user-images.githubusercontent.com/22956130/40759260-e6f4fa68-64ae-11e8-8e49-5921033e8804.png)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

### Log File

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS2.1.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
